### PR TITLE
EOS-25791 'hctl bootstrap' is not working on Hare kubernetes branch

### PR DESCRIPTION
**Pull request by SwapnilGaonkar7**
Starting consul as server.
Modified check-service to take host(hostname or IP address) as input

TODO: Test

### CODE REVIEW

File: utils/check-service [hunk 1]
```diff
@@ -117,7 +121,8 @@ if [[ $service =~ m0d ]]; then
     fi
 fi
 
-/opt/seagate/cortx/hare/bin/m0ping $(get_node_name) $port
+#/opt/seagate/cortx/hare/bin/m0ping $(get_node_name) $port
```
Comment (knekrasov) [inline utils/check-service]:
```suggestion
```

**Review by knekrasov — APPROVED** [review]

File: utils/check-service (hunk 1 above)
Comment (SwapnilGaonkar7) [reply utils/check-service]: Done

**Comment by mssawant** [comment]
@SwapnilGaonkar7, we need to try a custom deployment with this PR as it has changes to some related files.

**Comment by SwapnilGaonkar7** [comment]
3 node bootstrap is working,

[root@ssc-vm-g2-rhev4-2221 ~]# hctl bootstrap --mkfs /root/3node.yaml
2021-11-26 00:39:33: Generating cluster configuration... OK
2021-11-26 00:39:40: Starting Consul server on this node......... OK
2021-11-26 00:39:51: Importing configuration into the KV store... OK
2021-11-26 00:39:53: Starting Consul on other nodes...Consul ready on all nodes
2021-11-26 00:39:56: Updating Consul configuraton from the KV store... OK
2021-11-26 00:40:03: Waiting for the RC Leader to get elected.... OK
2021-11-26 00:40:06: Starting Motr (phase1, mkfs)... OK
2021-11-26 00:40:16: Starting Motr (phase1, m0d)... OK
2021-11-26 00:40:22: Starting Motr (phase2, mkfs)... OK
2021-11-26 00:40:32: Starting Motr (phase2, m0d)... OK
2021-11-26 00:40:39: Checking health of services... OK
[root@ssc-vm-g2-rhev4-2221 ~]# hctl status
Data pool:
    # fid name
    0x6f00000000000001:0x87 'the pool'
Profile:
    # fid name: pool(s)
    0x7000000000000001:0xcf 'default': 'the pool' None None
Services:
    ssc-vm-g2-rhev4-2222.colo.seagate.com
    [started]  hax        0x7200000000000001:0x32  inet:tcp:192.168.59.218@2001
    [started]  confd      0x7200000000000001:0x35  inet:tcp:192.168.59.218@3001
    [started]  ioservice  0x7200000000000001:0x38  inet:tcp:192.168.59.218@3002
    [unknown]  m0_client  0x7200000000000001:0x55  inet:tcp:192.168.59.218@5001
    [unknown]  m0_client  0x7200000000000001:0x58  inet:tcp:192.168.59.218@5002
    ssc-vm-g2-rhev4-2221.colo.seagate.com  (RC)
    [started]  hax        0x7200000000000001:0x6   inet:tcp:192.168.59.148@2001
    [started]  confd      0x7200000000000001:0x9   inet:tcp:192.168.59.148@3001
    [started]  ioservice  0x7200000000000001:0xc   inet:tcp:192.168.59.148@3002
    [unknown]  m0_client  0x7200000000000001:0x29  inet:tcp:192.168.59.148@5001
    [unknown]  m0_client  0x7200000000000001:0x2c  inet:tcp:192.168.59.148@5002
    ssc-vm-g2-rhev4-2223.colo.seagate.com
    [started]  hax        0x7200000000000001:0x5e  inet:tcp:192.168.51.59@2001
    [started]  confd      0x7200000000000001:0x61  inet:tcp:192.168.51.59@3001
    [started]  ioservice  0x7200000000000001:0x64  inet:tcp:192.168.51.59@3002
    [unknown]  m0_client  0x7200000000000001:0x81  inet:tcp:192.168.51.59@5001
    [unknown]  m0_client  0x7200000000000001:0x84  inet:tcp:192.168.51.59@5002

File: utils/update-consul-conf [hunk 2]
```diff
@@ -60,11 +60,12 @@ TEMP=$(getopt --options hns: \
 
 eval set -- "$TEMP"
 
-conf_dir=/var/lib/hare
+conf_dir=
 log_dir=/var/log/seagate/hare
 kv_file=/var/lib/hare/consul-kv.json
 dry_run=false
 server=false
+custom_config=true
```
[the remark this answers is not in the document]
Comment (SwapnilGaonkar7) [reply utils/update-consul-conf:73]: Done

**Review by vaibhavparatwar — APPROVED** [review]

**Comment by SwapnilGaonkar7** [comment]
LC deployment is done on 3node K8 setup

[root@cortx-data-headless-svc-ssc-vm-g3-rhev4-2653 /]# hctl status
Data pool:
    # fid name
    0x6f00000000000001:0x81 'storage-set-1__sns'
Profile:
    # fid name: pool(s)
    0x7000000000000001:0xcc 'Profile_the_pool': 'storage-set-1__sns' 'storage-set-1__dix' None
Services:
    cortx-data-headless-svc-ssc-vm-g3-rhev4-2655  (RC)
    [started]  hax        0x7200000000000001:0x31  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2655@2001
    [started]  ioservice  0x7200000000000001:0x34  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2655@3001
    [started]  ioservice  0x7200000000000001:0x41  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2655@3002
    [started]  confd      0x7200000000000001:0x4e  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2655@3003
    [started]  s3server   0x7200000000000001:0x51  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2655@4001
    [started]  s3server   0x7200000000000001:0x54  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2655@4002
    cortx-data-headless-svc-ssc-vm-g3-rhev4-2653
    [started]  hax        0x7200000000000001:0x7   inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2653@2001
    [started]  ioservice  0x7200000000000001:0xa   inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2653@3001
    [started]  ioservice  0x7200000000000001:0x17  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2653@3002
    [started]  confd      0x7200000000000001:0x24  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2653@3003
    [started]  s3server   0x7200000000000001:0x27  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2653@4001
    [started]  s3server   0x7200000000000001:0x2a  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2653@4002
    cortx-data-headless-svc-ssc-vm-g3-rhev4-2654
    [started]  hax        0x7200000000000001:0x5b  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2654@2001
    [started]  ioservice  0x7200000000000001:0x5e  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2654@3001
    [started]  ioservice  0x7200000000000001:0x6b  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2654@3002
    [started]  confd      0x7200000000000001:0x78  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2654@3003
    [started]  s3server   0x7200000000000001:0x7b  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2654@4001
    [started]  s3server   0x7200000000000001:0x7e  inet:tcp:cortx-data-headless-svc-ssc-vm-g3-rhev4-2654@4002

**Review by mssawant — APPROVED** [review]

**Pull request merged**